examples/caption_generation.ipynb

```diff
@@ -0,0 +1,1277 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    visualization_df = pd.DataFrame({'from':captions_df['filename'],'to':captions_df['filename'], 'label':captions_df['caption'], 'distance':0*len(captions_df),})
</pre><p> </p><p>As I know, we are trying to encourage users to use the V1 API. Not sure if using the old API would confuse users?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/visual-layer/fastdup/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I'm not sure, this is how it was suggested by Danny to do it, because I'm fairly certain the new API won't support this. Happy to address this and change some things, but marking the conversation as resolved for now.